lib/core/presentation/pages/notification/notifications_listing_page.dart

```diff
@@ -50,28 +53,30 @@ class _NotificationsListingViewState extends State<_NotificationsListingView> {
         title: t.notification.notifications,
         leading: BurgerMenu(),
         actions: [
-          ThemeSvgIcon(color: themeColor.onSurface, builder: (filter) => Assets.icons.icChat.svg(colorFilter: filter)),
+          ThemeSvgIcon(
+              color: themeColor.onSurface,
+              builder: (filter) =>
+                  Assets.icons.icChat.svg(colorFilter: filter)),
         ],
       ),
       backgroundColor: themeColor.primary,
       body: BlocBuilder<NotificationsListingBloc, NotificationsListingState>(
         builder: (context, state) {
           return state.when(
             loading: () => Loading.defaultLoading(context),
-            fetched: (filteredNotifications) {
+            fetched: (notifications) {
               return ListView.separated(
-                itemBuilder: (ctx, index) => index == filteredNotifications.length
-                    ? const SizedBox(height: 80)
-                    : NotificationCard(
-                        key: Key(filteredNotifications[index].id ?? ''),
-                        notification: filteredNotifications[index],
-                        onTap: () {
-                          NavigationUtils.handleNotificationNavigate(context, filteredNotifications[index]);
-                        },
-                      ),
+                itemBuilder: (ctx, index) => NotificationCard(
+                  key: Key(notifications[index].id ?? ''),
+                  notification: notifications[index],
+                  onTap: () {
+                    NavigationUtils.handleNotificationNavigate(
+                        context, notifications[index]);
+                  },
+                ),
                 separatorBuilder: (ctx, index) =>
                     SizedBox(height: Spacing.extraSmall),
-                itemCount: filteredNotifications.length + 1,
+                itemCount: notifications.length + 1,
```

Comment:
Maybe you misunderstand me, what I mean here is that we don't need to `plus 1` here any more :v

Reply:
Resolved